dct_backend/server_test.py

```diff
@@ -6,5 +6,15 @@
 def test_root():
     response = client.get("/")
     assert response.status_code == 200
-    assert response.json() == {"quack": "Hello World!"}
+    assert response.json() == {"msg": "This is the DCT Backend!"}
+
+
+data = {"stations": {"Stem To Root": {"name": "Stem To Root", "menu": [{"name": "Pancakes", "station": "Stem To Root", "meal_time": "Breakfast", "calories": "100", "categories": ["carbs"]}, {"name": "Wings", "station": "Stem To Root", "meal_time": "Lunch", "calories": "102", "categories": ["category1"]}, {"name": "Macaroni & Cheese", "station": "Stem To Root", "meal_time": "Dinner", "calories": "102", "categories": ["vegan", "carbs"]}, {"name": "Pasta", "station": "Stem To Root", "meal_time": "Breakfast", "calories": "100", "categories": ["carbs"]}], "line_speed": 0}, "605 Kitchen": {"name": "605 Kitchen", "menu": [{"name": "Bagel", "station": "605 Kitchen", "meal_time": "Lunch", "calories": "100", "categories": ["carbs"]}, {"name": "Grilled Chicken", "station": "605 Kitchen", "meal_time": "Dinner", "calories": "102", "categories": ["vegan", "carbs"]}, {"name": "Veggie Fried Rice", "station": "605 Kitchen", "meal_time": "Breakfast", "calories": "100", "categories": ["carbs"]}, {"name": "French Fries", "station": "605 Kitchen", "meal_time": "Lunch", "calories": "100", "categories": [
```

Comment:
These seem like lengthy tests. 
- How did you make them?
- This isn't hard-coded menu info, right?

Reply:
This is our mocked menu data put into explicit JSON format. So your webscraper should ideally return something similar to this. 

For tests, this is hardcoded